### DISCUSSION TITLE
Support Obsidian Timestamp Notes plugin

### DISCUSSION THREAD

**Discussion by cs-mshah** [alangrainger/share-note#29] [Vote for upcoming features]
The ability to have plugins enabled in shared notes would be great. Specifically, [Obsidian Timestamp Notes](https://github.com/juliang22/ObsidianTimestampNotes) would enable us to click on timestamps and play videos in the shared notes. Other reading view enhancement plugins would also be great.

**Comment by alangrainger** [comment]
I've renamed the title, as any plugin would have to be manually added. Many plugins could never function on the web since they rely on filesystem functions.

If enough people vote this up I can look into implementing support for Obsidian Timestamp Notes.
